benchmark/daisy/daisy.jl

```diff
@@ -0,0 +1,70 @@
+# ======================
+# Initialization
+# ======================
+
+# The benchmarks in this file are taken from
+# [Project Daisy](https://github.com/malyzajko/daisy/blob/master/testcases/).
+SUITE["Daisy"] = BenchmarkGroup()
+RESULTS = SUITE["Daisy"]
+
+# Dictionaries to store data and reference values
+DAISY_1D = Dict{String, Any}()
+DAISY_ND = Dict{String, Any}()
+
+include("setup.jl")
+include("univariate.jl")
+include("multivariate.jl")
+
+# ===============================
+# Run one-dimensional benchmarks
+# ===============================
+for name in keys(DAISY_1D)
+    RESULTS[name] = BenchmarkGroup()
+    RELPREC[name] = Dict()
+    dom = DAISY_1D[name]["dom"]
+    ref = DAISY_1D[name]["ref"]
+    for m in [2, 5, 10]
+        x = Taylor1(m)
+        p = DAISY_1D[name]["f"](x)
+
+        RESULTS[name]["order $m"] = BenchmarkGroup()
+        RELPREC[name]["order $m"] = Dict()
+
+        RESULTS[name]["order $m"]["evaluate"] = @benchmarkable evaluate($p, $dom)
```

Comment:
The benchmarks are testing the (polynomial) `evaluate`-ion of `p`, which is a `Taylor1` object. My point with this remark is that we are not evaluating a `TaylorModel1` object, but only the evaluation of its polynomial part. 

Rather than changing this, I would like to suggest to extend the benchmarks to `TaylorModel1` (and `TaylorModelN`) objects. Note that the same comment holds for the multivariate benchmarks below. Be aware that in the multivariate case the order has to be fixed differently in `set_variables`, I think to `2*m` (using your notation), due to the current algorithm used to bound the product of two Taylor Models.